format.py

```diff
@@ -87,7 +87,7 @@ def find_swiftformat(swift_format: str) -> Path:
 def get_files_to_format() -> List[Path]:
     package_dir = Path(__file__).parent
     files_to_format: List[Path] = []
-    for swift_file_path in (package_dir / "Sources").glob('**/*.swift'):
+    for swift_file_path in (package_dir).glob('**/Sources/**/*.swift'):
```

Comment:
Just a thought: Could we just do the following and then get rid of the `for` loop for `Tests`. AFAICT that would also format the examples, which would be good as well.

```suggestion
    for swift_file_path in package_dir.glob('**/*.swift'):
```